contrib/dynamic_embedding/.clang-format

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Lets put `https://github.com/pytorch/pytorch/blob/master/.clang-format` in the root of torchrec and have it as a main .clang-format for all torchrec and remove this file.